module/app.rb

```diff
@@ -0,0 +1,56 @@
+require 'colorize'
+require_relative './module/add_books'
+require_relative './module/list_items'
+
```

Comment:
I think I am facing an error in opening app because importing method is not correct here
![Screenshot from 2022-03-15 16-56-37](https://user-images.githubusercontent.com/51102661/158408500-2aab3ac2-fc89-4785-8436-23e6fdfe4259.png)